scanner/src/main/java/edu/ucr/cs/riple/scanner/Serializer.java

```diff
@@ -53,8 +53,8 @@ public class Serializer {
   public static final String METHOD_INFO_FILE_NAME = "method_info.tsv";
   public static final String CLASS_INFO_FILE_NAME = "class_info.tsv";
 
-  public Serializer(ErrorProneCLIFlagsConfig config) {
-    Path outputDirectory = config.outputDirectory;
+  public Serializer(Config config) {
+    Path outputDirectory = config.getOutputDirectory();
```

Comment:
https://github.com/nimakarimipour/NullAwayAnnotator/pull/19/files

Reply:
Sorry I didn't fully understand this comment. Would you please elaborate.